models/fct_dbt__exposures_updates.sql

```diff
@@ -0,0 +1,35 @@
+with model_updates as (
```

Comment:
A very minor point, but can we move `model_executions` and `dim_dbt_exposures` ref's into their own CTEs? This conforms to the [dbt style guide](https://github.com/fishtown-analytics/corp/blob/master/dbt_coding_conventions.md#ctes), and will eventually be [included in SQLFluff](https://github.com/sqlfluff/sqlfluff/issues/464) (the sql linter).

i.e.

```sql
with model_executions as (

    select * from {{ ref('fct_dbt__model_executions') }}

),

model_exposures as (

    select * from {{ ref('dim_dbt__exposures') }}

),

model_updates as (

    select 
         max(query_completed_at) as latest_update,
         node_id
     from model_executions
     group by node_id

),

...
```

Reply:
Thanks for pointing this out! I have refactored that model to include individual CTEs at the top as well as the final `select * from ` statement at the end.

Reply:
Excellent, thanks 🙏